olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -432,33 +432,33 @@ spec:
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-autoscaler"
                       - name: "IMAGE_autoscaler-hpa"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:knative-serving-autoscaler-hpa"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-autoscaler-hpa"
                       - name: "IMAGE_controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:knative-serving-controller"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-controller"
                       - name: "IMAGE_webhook__webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:knative-serving-webhook"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-webhook"
                       - name: "IMAGE_domain-mapping"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:knative-serving-domain-mapping"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-domain-mapping"
                       - name: "IMAGE_domainmapping-webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:knative-serving-domain-mapping-webhook"
-                      - name: "IMAGE_storage-version-migration-serving-serving-0.26.0__migrate"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:knative-serving-storage-version-migration"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-domain-mapping-webhook"
+                      - name: "IMAGE_storage-version-migration-serving-serving-1.0.1__migrate"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-storage-version-migration"
                       - name: "IMAGE_kourier-gateway"
                         value: "quay.io/maistra/proxyv2-ubi8:2.1.0"
                       - name: "IMAGE_kourier-control"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:kourier"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:kourier"
                       - name: "IMAGE_net-kourier-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:kourier"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.1:kourier"
                       - name: "IMAGE_net-istio-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.26.0:net-istio-controller"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.0.0:net-istio-controller"
```

Comment:
In the PR here https://github.com/openshift/release/pull/24708/files I have:
```
registry.ci.openshift.org/openshift/knative-v1.0.1:net-istio-controller quay.io/openshift-knative/net-istio-controller:v1.0.0
registry.ci.openshift.org/openshift/knative-v1.0.1:net-istio-webhook quay.io/openshift-knative/net-istio-webhook:v1.0.0
```
Is this ok? @markusthoemmes . It seems images resolve though.

Reply:
If it works, sure.

Reply:
I guess it doesn't work though. The image mirroring job keeps failing, see https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-image-mirroring-knative/1471888035317551104, which causes some images to not be available. Relevant bit:

```
error: unable to retrieve source image registry.ci.openshift.org/openshift/knative-v1.0.1 by tag net-istio-controller: manifest unknown: manifest unknown
error: unable to retrieve source image registry.ci.openshift.org/openshift/knative-v1.0.1 by tag net-istio-webhook: manifest unknown: manifest unknown 
```

Reply:
@skonto the promotion setting specifies where the image is pushed. For net-istio,  [you've configured `knative-v1.0.0`](https://github.com/openshift/release/blob/ef6b39edf3d78499836ef19969f6c99f47c79231/ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.0.yaml#L33) while for net-kourier you specified [`knative-v1.0.1`](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.0.yaml#L23). That's generally fine, but the image mirror has to be setup correctly as well, which it currently isn't (net-istio tries to fetch knative-v1.0.1 which doesn't exist per the settings above).

Reply:
In this case, a fix would be to change the former reference in https://github.com/openshift/release/blob/ef6b39edf3d78499836ef19969f6c99f47c79231/core-services/image-mirroring/knative/mapping_knative_v1_0_quay#L41-L42 to `knative-v1.0.0` instead of `knative-v1.0.1`

Reply:
Yeah makes sense will fix shortly.